server/serverimpl.go

```diff
@@ -264,6 +264,10 @@ func (s *server) handleWSConnection(reqCtx context.Context, wsConn *websocket.Co
 		}
 
 		response := connectionCallbacks.OnMessage(msgContext, agentConn, &request)
```

Comment:
Please also update doc-comment for OnMessage. It currently says "Must return a ServerToAgent message...".

Reply:
Done.